core/trie/src/lib.rs

```diff
@@ -520,7 +520,7 @@ mod tests {
 			memtrie.commit();
 			if *memtrie.root() != real {
 				println!("TRIE MISMATCH");
-				println!("");
+				println!();
```

Comment:
```suggestion
				println!("");
```

Reply:
Why?

Reply:
It aligns nicely and looks less weird.
A better solution would be to print everything at once though, rather than calling `println!` multiple times.